plugins/polio/migrations/0149_create_reasons_for_delay.py

```diff
@@ -0,0 +1,69 @@
+from django.db import migrations, models
+
+from plugins.polio.models import DelayReasons
+
+TRANSLATIONS = {
+    "INITIAL_DATA": {"en": "Initial data", "fr": "Données initiales"},
+    "ENCODING_ERROR": {"en": "Encoding error", "fr": "Erreur d'encodage"},
+    "PUBLIC_HOLIDAY": {"en": "Delayed to observe public holiday", "fr": "Retardé pour observer un jour férié"},
+    "OTHER_ACTIVITIES": {
+        "en": "Integrate with other vaccination activities",
+        "fr": "Intégration à d'autres activités de vaccination",
+    },
+    "MOH_DECISION": {"en": "Decision from MOH", "fr": "Décision du Ministère de la Santé"},
+    "CAMPAIGN_SYNCHRONIZATION": {"en": "Campaigns synchronization", "fr": "Synchonisation des campagnes"},
+    "PREPAREDNESS_LEVEL_NOT_REACHED": {
+        "en": "Preparedness level not reached",
+        "fr": "Niveau de préparation non-atteint",
+    },
+    "FUNDS_NOT_RECEIVED_OPS_LEVEL": {
+        "en": "Funds not received at operational level",
+        "fr": "Fonds non reçus au niveau opérationnel",
+    },
+    "FUNDS_NOT_ARRIVED_IN_COUNTRY": {"en": "Funds not arrived in country", "fr": "Fonds non arrivés dans le pays"},
+    "VACCINES_NOT_DELIVERED_OPS_LEVEL": {
+        "en": "Vaccines not delivered at operational level",
+        "fr": "Vaccins non délivrés au niveau opérationnel",
+    },
+    "VACCINES_NOT_ARRIVED_IN_COUNTRY": {
+        "en": "Vaccines not arrived in country",
+        "fr": "Vaccins non arrivés dans le pays",
+    },
+    "SECURITY_CONTEXT": {"en": "Security context", "fr": "Contexte sécuritaire"},
+    "CAMPAIGN_MOVED_FORWARD_BY_MOH": {
+        "en": "Campaign moved forward by MOH",
+        "fr": "Campagne avancée par le Ministère de la Santé",
+    },
+    "VRF_NOT_SIGNED": {"en": "VRF not signed", "fr": "VRF non signée"},
+    "FOUR_WEEKS_GAP_BETWEEN_ROUNDS": {
+        "en": "Four weeks gap between rounds to be respected",
+        "fr": "Intervalle de 4 semaines entre les rounds à respecter",
+    },
+    "OTHER_VACCINATION_CAMPAIGNS": {"en": "Other vaccination campaigns", "fr": "Autres campagnes de vaccination"},
+}
+
+
+def create_reasons_for_delay(apps, schema_editor):
+    ReasonForDelay = apps.get_model("polio", "ReasonForDelay")
+    text_choice_keys = [choice.name for choice in DelayReasons]
+    for choice_key in text_choice_keys:
```

Comment:
```suggestion
    for choice_key in DelayReasons.values:
```

This would give you the same result.